src/admin/dumb_components/sub_due_form.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+const { reduxForm } = require('redux-form')
+
+const sub_due_dates = ({fields: {start, end}, handleSubmit, error}) =>
+  <form className='date-boundaries-form' onSubmit={handleSubmit}>
+    <input type='text' placeholder='Start date' {...start}/>
+    {start.touched && start.error && <div>{start.error}</div>}
+    <input type='text' placeholder='End date' {...end}/>
+    {start.touched && start.error && <div>{start.error}</div>}
```

Comment:
Also this functionality is already abstracted into `src/admin/dumb_components/field.js` (sorry, should have mentioned earlier)